pkg/grpc/api.go

```diff
@@ -594,3 +597,15 @@ func emitACLMetrics(actionPolicy, appID, trustDomain, namespace, operation, verb
 		}
 	}
 }
+
+func (a *api) SetAppChannel(appChannel channel.AppChannel) {
+	a.appChannel = appChannel
+}
+
+func (a *api) SetDirectMessaging(directMessaging messaging.DirectMessaging) {
+	a.directMessaging = directMessaging
+}
+
+func (a *api) SetActor(actor actors.Actors) {
```

Comment:
Please use `SetActorRuntime` instead of `SetActor`

actors.Actors is not just actor. it is actor runtime. We will need to rename this later but for now. please use SetActorRuntime.